build_scripts/windows/azure-cli.wixproj

```diff
@@ -21,6 +12,15 @@
     <WixTargetsPath Condition=" '$(WixTargetsPath)' == '' AND '$(MSBuildExtensionsPath32)' != '' ">$(MSBuildExtensionsPath32)\Microsoft\WiX\v3.x\Wix.targets</WixTargetsPath>
     <WixTargetsPath Condition=" '$(WixTargetsPath)' == '' ">$(MSBuildExtensionsPath)\Microsoft\WiX\v3.x\Wix.targets</WixTargetsPath>
   </PropertyGroup>
+  <!-- Local WiX -->
+  <PropertyGroup>
+    <LocalWixRoot>artifacts\wix</LocalWixRoot>
+    <WixToolPath>$(LocalWixRoot)</WixToolPath>
+    <WixTargetsPath Condition="Exists('$(WixToolPath)\Wix.targets')">$(WixToolPath)\Wix.targets</WixTargetsPath>
+    <WixTasksPath Condition="Exists('$(WixToolPath)\wixtasks.dll')">wixtasks.dll</WixTasksPath>
+    <AzureCliSource>artifacts\cli</AzureCliSource>
+    <LinkerAdditionalOptions>-fv</LinkerAdditionalOptions>
+  </PropertyGroup>
```

Comment:
I am a little bit curious about why this `PropertyGroup` is moved here.

Reply:
To pick up the properties I set above. Could be all in the same PropertyGroup, but I left them separate with your comments for easier identification as to what they do.